app/controllers/api/notes_controller.rb

```diff
@@ -20,7 +20,7 @@ def feedback_mail_params
         :os,
         :browser,
         :device,
-        :version,
+        #        :version,
```

Comment:
this was ignored.  so took it out here.   maybe we just dump it entirely.  just wasn't sure yet